pages/what-we-do/partnerships/communities.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Digital communities of practice
+layout: basic
+permalink: /what-we-do/partnerships/communities/
```

Comment:
Add redirects from the old page:
/what-we-do/communities/
/our-work/communities/